en-2016-07-17-latex-template-for-responses-to-referees-reports/statement.tex

```diff
@@ -1,63 +1,64 @@
-%
 % Title:    A LaTeX Template For Responses To a Referees' Reports
 % Author:   Petr Zemek <s3rvac@gmail.com>
 % Homepage: https://blog.petrzemek.net/2016/07/17/latex-template-for-responses-to-referees-reports/
 % License:  CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
-%
-\documentclass[a4paper,10pt]{article}
+\documentclass[10pt]{article}
 
-% Packages
+% Allow Unicode input (alternatively, you can use XeLaTeX or LuaLaTeX)
 \usepackage[utf8]{inputenc}
 
-% View
-\setlength{\hoffset}{-1.5cm}
-\setlength{\voffset}{-1.5cm}
-\setlength{\textheight}{23.0cm}
-\setlength{\textwidth}{15.2cm}
-
-% Paragraph formatting (no indentation)
-\setlength{\parskip}{1.3ex plus 0.2ex minus 0.2ex}
-\setlength{\parindent}{0pt}
-
-% Environments
-\newenvironment{comment}
-	{\begin{tabular}{|p{\textwidth}}\begin{tt}}
```

Comment:
`tt` isn't an environment – this only works because `\begin{myenv}` calls `\myenv`. In this case, `\begin{tt}` calls `\tt`, a deprecated command.